Circular_kernel_2/include/CGAL/Circular_kernel_2/Circular_arc_2.h

```diff
@@ -191,7 +191,7 @@ namespace internal {
 	// the circles intersect
 	
 	const std::pair<typename CK::Circular_arc_point_2, unsigned>*
-          result = CGAL::internal::intersect_get< std::pair<typename CK::Circular_arc_point_2, unsigned> >(*it);
+          result = CGAL::Intersections::internal::intersect_get< std::pair<typename CK::Circular_arc_point_2, unsigned> >(*it);
```

Comment:
At the next developers meeting we should talk about that. Our coding guidelines are currently wrong about the use of `internal` as a namespace: they say `CGAL::internal::Intersections` should be used, but I disagree, and it seems you as well.